lib/matplotlib/axes/_axes.py

```diff
@@ -3555,7 +3561,6 @@ def bxp(self, bxpstats, positions=None, widths=None, vert=True,
         # empty list of xticklabels
         datalabels = []
 
-        zorder = mlines.Line2D.zorder
```

Comment:
I have a weak preference for this continuing to track `mlines.Line2D.zorder` by default, but could very easily be talked into change it.